src/EventStore.ClusterNode/ClusterVNodeHostedService.cs

```diff
@@ -102,6 +102,11 @@ internal class ClusterVNodeHostedService : EventStoreHostedService<ClusterNodeOp
 			if (opts.EnableAtomPubOverHTTP) {
 				Log.Warning("\n DEPRECATION WARNING: AtomPub over HTTP Interface has been deprecated in version 20.02. It is recommended to use GRPC instead.\n");
 			}
+			if (!opts.EnableExternalTCP) {
```

Comment:
This seems backwards? We should display the message if it _is_ enabled, rather than if it is not surely?

Reply:
Agreed, as discussed I will rework the deprecation warnings all together.